.github/stale.yml

```diff
@@ -0,0 +1,62 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 180
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Only issues or pull requests with all of these labels are check if stale. Defaults to `[]` (disabled)
+onlyLabels: []
+
+# Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
+exemptLabels:
+  - pin
+  - "feature request"
+  - "eng:disabled-test"
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: wontfix
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+
+# Comment to post when removing the stale label.
+# unmarkComment: >
+#   Your comment here.
+
+# Comment to post when closing a stale Issue or Pull Request.
+# closeComment: >
+#   Your comment here.
+
+# Limit the number of actions per hour, from 1-30. Default is 30
+limitPerRun: 30
+
+# Limit to only `issues` or `pulls`
+only: issues
+
+# Optionally, specify configuration settings that are specific to just 'issues' or 'pulls':
+# pulls:
+#   daysUntilStale: 30
+#   markComment: >
+#     This pull request has been automatically marked as stale because it has not had
+#     recent activity. It will be closed if no further activity occurs. Thank you
+#     for your contributions.
+
+issues:
```

Comment:
At [lines 14-17](https://github.com/mozilla-mobile/focus-android/pull/6805/files#diff-76cc7f7adb754443d391eb9c8df1d4a47e8cd668179e41915153685a0b83b30aR14-R17) there is a list of labels that will prevent issues/PRs from being considered stale.
Then at [line 49](https://github.com/mozilla-mobile/focus-android/pull/6805/files#diff-76cc7f7adb754443d391eb9c8df1d4a47e8cd668179e41915153685a0b83b30aR49) this script is configured to only apply to issues.

As such we probably don't need this block for a issues-only configuration.

There is one difference though regarding `exemptLabels` - `eng:disabled-test`. Maybe someone from QA can chime in about if they want the stale bot to also close those tickets or not.

Reply:
@sv-ohorvath @AndiAJ ^^

Reply:
@ionutbedregeanu no, please don't close eng:disabled-test issues